pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -247,22 +248,21 @@ public static Address addParticipantAddressFromFile(String esIndex, String fileN
     /**
      * Add a DSM entity to the participant doc
      *
-     * @param dob date of birth to replace in DSM entity
+     * @param dob            date of birth to replace in DSM entity
```

Comment:
Does something in you IDE make these formatting changes, or is it something you do by hand?

Reply:
This is all checkstyle

Reply:
Checkstyle is doing the reformatting automatically, or complaining about the indenting?
I don't see that in my IDE, and usually it shows checkstyle violations.

![image](https://github.com/broadinstitute/ddp-study-server/assets/129115982/3f3a22e7-20cc-435c-9aab-9a629f1ab3fa)